.github/workflows/ci_tests_dev.yaml

```diff
@@ -0,0 +1,79 @@
+# This workflow installs PyGMT dependencies, builds documentation and runs tests on GMT master
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: GMT Master Tests
+
+on:
+  # push:
+  #   branches: [ master ]
+  pull_request:
+    types: [review_requested, ready_for_review]
+  # Schedule daily tests
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  test_gmt_master:
+    name: ${{ matrix.os }} - Python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.8]
+        os: [ubuntu-20.04, macOS-10.15]
```

Comment:
Would love to test on Windows here, but I think we might have to do https://github.com/conda-forge/gmt-feedstock/issues/95 :wink: